### ISSUE TITLE
Model Inference time

### ISSUE THREAD

**Issue by hnzjuer** [Malikanhar/Face-Attribute-Classification#1]
Hi, thank you for your code really!. 
I used a pre-trained model for testing. 
The accuracy is high, but the inference time for each picture is very long. 
The model.predict() step took about 24s for each picture. 
This phenomenon is very strange. 
Can you provide some help?

**Comment by atomicsyed-99** [comment]
Hey! , this repository is really amazing , but the links to the pre-trained model dont seem to be working! , any help with that?

**Comment by Peregalli** [comment]
Hi! Would you upload pre-trained model again? Link doesnt work :(

**Comment by calleysperx** [comment]
Would you upload pre-trained model again? Link doesnt work :(

**Comment by JavanehBahrami** [comment]
Hi @Malikanhar 
Would you upload pre-trained model again? Link doesnt work :(

**Comment by Malikanhar** [comment]
Hi @hnzjuer are you using GPU to run the model? And could you give me the details of processing that you use for running the model?

**Comment by Malikanhar** [comment]
I stored the pre-trained model on my student drive, which has since expired, and I don't have a backup. Nonetheless, you can use my training scripts to train the model from scratch, and it should function properly.
